gateway/shard/createShard.ts

```diff
@@ -91,6 +90,7 @@ export function createShard(
     state: ShardState.Offline,
     /** The total amount of shards which are used to communicate with Discord. */
     totalShards: options.totalShards,
+    resumeGatewayUrl: "",
```

Comment:
```suggestion
    resumeGatewayUrl: undefined as string | undefined,
```

Reply:
@itohatweb skillz made me change that from `undefined as string | undefined` to `""`
@Skillz4Killz why did you mark this as resolved

Reply:
why